installer/lib/installer.py

```diff
@@ -95,39 +72,27 @@ def bootstrap(self, verbose: bool = False) -> TemporaryDirectory:
         cmd.extend(self.reqs)
 
         try:
+            # upgrade pip to the latest version to avoid a confusing message
+            res = upgrade_pip(Path(venv_dir.name))
+            if verbose:
+                print(res)
+
+            # run the install prerequisites installation
             res = subprocess.check_output(cmd).decode()
+
             if verbose:
                 print(res)
+
             return venv_dir
         except subprocess.CalledProcessError as e:
             print(e)
```

Comment:
What does it mean if this fails? It looks like we attempt an install even if there is an error

Reply:
it means bootstrapping of the installer failed, so it won't be able to install anyway. But yes, we could add a `sys.exit(1)` here for a cleaner exit.